packages/settings-library/src/settings_library/tracing.py

```diff
@@ -7,14 +7,15 @@
 
 class TracingSettings(BaseCustomSettings):
     TRACING_ZIPKIN_ENDPOINT: AnyUrl = Field(
-        "http://jaeger:9411", description="Zipkin compatible endpoint"
+        default=parse_obj_as(AnyUrl, "http://jaeger:9411"),  # NOSONAR
+        description="Zipkin compatible endpoint",
     )
     TRACING_THRIFT_COMPACT_ENDPOINT: AnyUrl = Field(
-        "http://jaeger:5775",
+        default=parse_obj_as(AnyUrl, "http://jaeger:5775"),  # NOSONAR
```

Comment:
I think you can also log in sonar and say there won't fix with the reason.
Advantage is that if we ever change the value in here, the sonar will complain again where here it is silent forever.

Reply:
good point